versioned_docs/version-5/url.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Share machines using their share URL
+---
+
+You can share machines with your team or publicly with the machine URL.
+
+Your machine will only be viewable if:
+
+- the project visibility is **public**
+- the project visibility is **unlisted**
+- the project visibility is **private**, and the viewer is also a member of your team
```

Comment:
I now see this example with 3 bullet points which makes perfect sense. Maybe we should list bullet points for 2 or more items (in reference to my other comment).

Reply:
Yeah, I’m not sure how one bullet point happened. I like bullets for breaking information into a more skimmable format, it also serves to mix up the formatting a bit so it’s not endless paragraphs!